src/data/actions/grades.test.js

```diff
@@ -176,4 +180,92 @@ describe('actions', () => {
       });
     });
   });
+
+  describe('fetchGradeOverridHistory', () => {
+    const subsectionId = 'subsectionId-11111';
+    const userId = 'user-id-11111';
+
+    const fetchOverridesURL = `${LmsApiService.baseUrl}/api/grades/v1/subsection/${subsectionId}/?user_id=${userId}&history_record_limit=5`;
+
+    const originalGrade = {
+      earned_all: 1.0,
+      possible_all: 12.0,
+      earned_graded: 3.0,
+      possible_graded: 8.0,
+    };
+
+    const override = {
+      earned_all_override: 13.0,
+      possible_all_override: 13.0,
+      earned_graded_override: 10.0,
+      possible_graded_override: 10.0,
+    };
+
+    it('dispatches success action after successfully getting override info', () => {
+      const responseData = {
+        success: true,
+        original_grade: originalGrade,
+        history: [],
+        override,
+      };
+
+      axiosMock.onGet(fetchOverridesURL)
+        .replyOnce(200, JSON.stringify(responseData));
+
+      const expectedActions = [
+        {
+          type: GOT_GRADE_OVERRIDE_HISTORY,
+          overrideHistory: [],
+          currentEarnedAllOverride: override.earned_all_override,
+          currentPossibleAllOverride: override.possible_all_override,
+          currentEarnedGradedOverride: override.earned_graded_override,
+          currentPossibleGradedOverride: override.possible_graded_override,
+          originalGradeEarnedAll: originalGrade.earned_all,
+          originalGradePossibleAll: originalGrade.possible_all,
+          originalGradeEarnedGraded: originalGrade.earned_graded,
+          originalGradePossibleGraded: originalGrade.possible_graded,
+        },
+      ];
+      const store = mockStore();
+
+      return store.dispatch(fetchGradeOverrideHistory(subsectionId, userId)).then(() => {
+        expect(store.getActions()).toEqual(expectedActions);
+      });
+    });
+
+    describe('dispatches failure action with expected message', () => {
+      it('on failure response', () => {
+        const responseData = {
+          success: false,
+          error_message: 'There was an error!!!!!!!!!',
+        };
+
+        axiosMock.onGet(fetchOverridesURL).replyOnce(200, JSON.stringify(responseData));
+
+        const expectedActions = [{
+          type: ERROR_FETCHING_GRADE_OVERRIDE_HISTORY,
+          errorMessage: responseData.error_message,
+        }];
+        const store = mockStore();
+
+        return store.dispatch(fetchGradeOverrideHistory(subsectionId, userId)).then(() => {
+          expect(store.getActions()).toEqual(expectedActions);
+        });
+      });
+
+      it('on 500 error', () => {
```

Comment:
nit: I would use `test('on 500 error')` or describe more in the title what the expected behavior is.